scipy/_lib/_util.py

```diff
@@ -397,3 +397,63 @@ def __call__(self, func, iterable):
             # wrong number of arguments
             raise TypeError("The map-like callable must be of the"
                             " form f(func, iterable)")
+
+
+# _normalize_axis_index() is a reimplementation of the private numpy function
+# numpy.core.multiarray.normalize_axis_index.  If that function is made public,
+# it can be used instead of this one.
+
+def _normalize_axis_index(axis, ndim, msg_prefix=None):
```

Comment:
Should we be adding types for new functions like these? This one in particular has very straightforward typing.

Reply:
Sure, but the question is moot, as I'm going to use the numpy function instead of this one.